lib/park.rb

```diff
@@ -0,0 +1,43 @@
+class Park 
+  attr_reader :name, :admission_price, :vehicles, :passengers
```

Comment:
```suggestion
  attr_reader :name, :admission_price, :vehicles
```

There is no `@passengers` attribute so we don't need an attr_reader.